src/controllers/User.ts

```diff
@@ -57,18 +66,22 @@ export async function getById (req: Request, res : Response){
         console.error(e);
     }
 }
-export async function destroy (req : Request, res : Response) {
-    try{
+export async function destroy(req: Request, res: Response) {
+    try {
         const client = await pool.connect();
-        const id : number = Number(req.params.id);
+        const id: number = Number(req.params.id);
         const result = await client.query("DELETE FROM users WHERE id = $1", [id]);
+
+        if (result.rowCount === 0) {
+            res.status(404).json({ message: "User not found" });
+        } else {
+            res.status(200).json({ message: "User deleted successfully" });
+        }
+
         client.release();
-        console.log(result.rows);
-        res.json(result.rows);
-    }
-    catch(e)
-    {
+    } catch (e) {
```

Comment:
When user is deleted, the user's profile picture (file) also needed to be deleted if exists.